.npmignore

```diff
@@ -0,0 +1,20 @@
+*.rb
+*.rake
+*.erb
+*.yml
+*.adoc
+*.txt
+**/.keep
+**/bin/
+**/spec/
+*.gemspec
+Gemfile
+Rakefile
+*.md
+*.gem
+*.tgz
+*.lock
+coverage/
+decidim_app-design/
+docs/
+development_app/
```

Comment:
I think you could just add an empty [`files` field](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#files) in the `package.json` to avoid including everything in the NPM package.

I think the package does not really have to include anything else than the dependencies at this point.

Reply:
That's much better, thanks! ❤️